src/EA.Weee.Web/Areas/Scheme/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -327,6 +327,21 @@ public async Task<ActionResult> DownloadEvidenceNotesReport(Guid pcsId, int comp
             }
         }
 
+        [HttpGet]
+        public async Task<ActionResult> DownloadEvidenceTransfersReport(Guid pcsId, int complianceYear)
+        {
+            using (var client = apiClient())
```

Comment:
@GuidoWorkDev unit tests :)

Reply:
I have added unit tests to cover this code